node_lambnik/scripts/publish

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+function usage() {
+    echo -n "Usage: $(basename "${0}") [OPTION]
+Login to a running Docker container's shell.
+Options:
+    --new     Deploy as a new lambda
+    --help      Display this help text
+"
+}
+
+function main() {
+    if [ "${1}" = "--new" ]
+    then
+        docker-compose run tiler bash -c "yarn deploy-new"
+    else
+        docker-compose run tiler bash -c "yarn deploy"
```

Comment:
I still don't understand why `./scripts/publish` just returns "success Already up to date" for me (per the screenshot earlier) but if I change these to just run yarn rather than running bash which runs yarn, i.e. `docker-compose run tiler yarn deploy-new` and `docker-compose run tiler yarn deploy`, it works.

Reply:
That's super strange, I haven't encountered that behavior at all. Have you tried doing `docker-compose down` just to clear everything before you update?

Reply:
Yeah, no change from clearing everything and building again.  I wonder if there's a bug in my version of `docker-compose`.  It seems like it's dropping the argument and just running `yarn`.  But it works fine if I run it in the container:
![image](https://user-images.githubusercontent.com/6598836/41609965-f541935a-73b9-11e8-96e2-ee119a6ea4bc.png)

What version of docker-compose do you have?  Mine says it's 1.17.1.
Also, do the `publish` commands that work for me work for you?  If so, we might as well do without the extra layer of `bash -c`...